test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1

```diff
@@ -114,5 +114,12 @@ Describe "Stream writer tests" -Tags "CI" {
             $result.Count | Should -Be $returnCount
             (Compare-Object $result $returnValue -SyncWindow 0).length | Should -Be 0
         }
+
+        It "Write-Information accepts `$Null" {
+            $streamPath = Join-Path $testdrive information.txt
+            $null | Write-Information -Tags myTag -ErrorAction Stop -InformationAction SilentlyContinue -InformationVariable i
+            $i.Tags | Should -BeExactly "myTag"
+            $i.MessageData | Should -BeNullOrEmpty
```

Comment:
Is there a way to test that this is just `null` or just empty? If there is any semantic difference for PowerShell's purposes, we should test specifically for the right one so that we can capture regressions.

Reply:
In this case, there's no difference.  The "OrEmpty" as it applies to this Pester assertion is only validating if a string is [string]::IsNullOrEmpty() or if a collection if count == 0.  Otherwise, it's checking if it's $null.